src/PaxfulApi.ts

```diff
@@ -2,14 +2,8 @@ import { Http2ServerResponse } from "http2";
 
 import { Profile, Credentials, CredentialStorage } from "./oauth";
 import { ApiConfiguration } from "./ApiConfiguration";
-import {
-    authorize,
-    retrieveImpersonatedCredentials,
-    retrievePersonalCredentials,
-    getProfile,
-    invoke,
-    InvokeBody
-} from "./commands";
+import { authorize, retrieveImpersonatedCredentials, retrievePersonalCredentials, getProfile, executeRequestAuthorized } from "./commands";
+import { AnyJson, InvokeBody, RequestBuilder, RequestResponse } from "./commands/Invoke";
```

Comment:
Are you sure about `RequestResponse` name?

Reply:
Changed to AnyPromise